seqr/views/apis/report_api.py

```diff
@@ -70,10 +74,10 @@ def seqr_stats(request):
     })
 
 
-def _get_sample_counts(sample_q):
-    samples_agg = sample_q.filter(is_active=True).values('sample_type', 'dataset_type').annotate(count=Count('*'))
+def _get_sample_counts(sample_q, data_type_key='dataset_type'):
```

Comment:
I was about to get confused about data_type vs dataset_type - but I imagine that it was a decision you made to use data_type the RNA samples because to keep the 2 concepts separate, and I think I agree with it

Reply:
yeah I went back and forth a couple times but ultimately felt like having it named something different made it a tiny bit clearer that its not the same thing